### DISCUSSION TITLE
TFLite

### DISCUSSION THREAD

**Discussion by mnznan** [RapidAI/RapidLatexOCR#13] [Q&A] [answered]
Hello, is there any chance of a sample on how to export the model to tflite?

thanks!

**Comment by SWHL** [comment, marked as answer]
I haven't tried this conversion.

You can refer to the solutions under the following:
- [PyTorch-ONNX-TFLite](https://github.com/sithu31296/PyTorch-ONNX-TFLite)
- [How do you convert a .onnx to tflite?](https://stackoverflow.com/questions/53182177/how-do-you-convert-a-onnx-to-tflite)

Hope it helps you.
